### DISCUSSION TITLE
Too many releases

### DISCUSSION THREAD

**Discussion by schrepfler** [dotansimha/graphql-code-generator#7330] [General]
While I really appreciate each artefact has it's own version and release notes, I can't say I'm a fan of the big batch of releases hitting my inbox all at once. If this is an atomic process, why not collapse them all on a single release page? (I'm aware people might feel differently about this and I understand I can unsubscribe or filter out the email notification but still, just asking a question)

**Comment by Urigo** [comment]
Hi @schrepfler yes I agree it's hard to see what were the recent changes on the Github release page with so many releases.
We are using the changeset tool, do you think you can help us research a good approach to keep the automation of the releases but also to make them more readable?

**Reply by lanwin** [reply]
I really would like to see a solution to this too. Cause currently most realeases flooding my inbox too.

**Comment by schrepfler** [comment]
Hi @Urigo, if the process of tagging is atomic, even if every artefact has different versions, perhaps the release step can generate one release and a single changelog with all the changeset embedded one behind the other as different paragraphs?

**Reply by Urigo** [reply]
any chance you can help us find the best solution?  the way we manage the repo is also open source and we would love help with that: https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md
